README.rst

```diff
@@ -0,0 +1,54 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2022 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+==============================
+ Cookiecutter-Invenio-ILS
+==============================
+
+.. image:: https://img.shields.io/github/license/inveniosoftware/cookiecutter-invenio-ils.svg
+        :target: https://github.com/inveniosoftware/cookiecutter-invenio-ils/blob/master/LICENSE
+
+.. image:: https://img.shields.io/travis/inveniosoftware/cookiecutter-invenio-ils.svg
+        :target: https://travis-ci.org/inveniosoftware/cookiecutter-invenio-ils
```

Comment:
We don't use `Travis` anymore (it is apparently wrong in RDM too!), but GitHub CI instead: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge
Can you please fix RDM too?